### PR TITLE
[REEF-1240] Fix CanRunClrBridgeExampleOnLocalRuntime failure

### PR DESCRIPTION
**Pull request by anupam128**
* Fix the behavior of LocalClient.SubmitAndGetJobStatus to start Java in
 * background
 * Add logging helper to async utils to log on failure of fire-and-forget
 * tasks

JIRA:
  [REEF-1240](https://issues.apache.org/jira/browse/REEF-1240)

### CODE REVIEW

File: lang/cs/Org.Apache.REEF.Utilities/AsyncUtils/LoggingHelper.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System.Threading.Tasks;
+using Org.Apache.REEF.Utilities.Logging;
+
+namespace Org.Apache.REEF.Utilities.AsyncUtils
+{
+    public static class LoggingHelper
```
Comment (markusweimer) [inline lang/cs/Org.Apache.REEF.Utilities/AsyncUtils/LoggingHelper.cs]: Please document this class and its methods.

**Comment by markusweimer** [comment]
@anupam128 I'm done with a pass.

**Comment by anupam128** [comment]
@markusweimer made the change. Please take a look.

**Comment by markusweimer** [comment]
LGTM, will test and merge

**Comment by markusweimer** [comment]
I merged this. The mirror seems to be stuck again.

**Pull request closed without merging**